source-firestore/pull.go

```diff
@@ -260,10 +260,10 @@ func computeBackfillStartTime(cfg *config, res *resource, prevBackfill *backfill
 	// - Advanced Endpoint Config
 	// - Default to 24h if no cursor or 5m if there's a cursor
 	var delay time.Duration
-	if d, err := time.ParseDuration(res.MinBackfillInterval); err == nil && d > 0 {
-		delay = d
-	} else if d, err := time.ParseDuration(cfg.Advanced.MinBackfillInterval); err == nil && d > 0 {
-		delay = d
+	if res.MinBackfillInterval != "" {
+		delay, _ = time.ParseDuration(res.MinBackfillInterval) // Cannot fail, already validated
```

Comment:
this is somewhat unsafe, as it doesn't account for changes made to validation logic and the propagation delay as tasks are re-published, and it would fail "open" in that it backfills over and over as quickly as possible.

Probably better to panic here, if we want to avoid threading through an error.

Reply:
I'm not sure I understand, either there's a correct value for the appropriate property name (in which case we're fine) or there's a non-empty and incorrect value, in which case `(resource).Validate()` will fail when it's called via `UnmarshalStrict()` during the task startup?

But sure, I'm happy to be more defensive here just in case.

Reply:
Ah, I missed that Validate is called on every startup and not just when driven by a control plane Validate RPC.
I retract this comment.